adcm_aio_client/core/objects/cm.py

```diff
@@ -232,9 +234,29 @@ def get_own_path(self: Self) -> Endpoint:
         return self.PATH_PREFIX, self.id
 
 
-class HostsNode(PaginatedAccessor[Host, None]):
+class HostsNode(PaginatedAccessor[Host, dict | None]):
     class_type = Host
 
 
-class HostsInClusterNode(PaginatedAccessor[Host, None]):
-    class_type = Host
+class HostsInClusterNode(HostsNode):
```

Comment:
hosts node should have create function, I believe, but hosts in cluster shouldn't

Reply:
renamed `HostsNode` to `HostsAccessor`